### ISSUE TITLE
Problem with icons in st

### ISSUE THREAD

**Issue by schvabodka-man** [sebastiencs/icons-in-terminal#9]
There is a rendering problem in suckless st.
On the left there is st, on the right xfce4-term(libvte based)
![alt text](https://transfer.sh/eh5M5/Screenshot_2017-10-27_10-48-23.png)
![alt text](https://transfer.sh/rTCSJ/Screenshot_2017-10-27_10-50-05.png)
The shell is fish if that matters

**Comment by thomas-louvigne** [comment]
We can't see your image :/

But it work perfectly at home ! (i am on zsh)

![2018-02-28-111608_3600x1080_scrot](https://user-images.githubusercontent.com/145759/36902711-bd228cd8-1e2b-11e8-9e6a-20418225725a.png)

**Comment by schvabodka-man** [comment]
Yep, the images are too old and they was removed most likely. I'll try to play a little bit with st config.
